<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrafficAnalysis</class>
 <widget class="QWidget" name="TrafficAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>491</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>418</width>
    <height>491</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Анализатор трафика</string>
  </property>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	
	background-color: rgb(0, 0, 127);
	color: white;
	border: 1px solid gray;
}
QPushButton:pressed
{
	background-color: rgb(0, 85, 127);
}</string>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_find">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgb(70,130,180);
	color: white;
	border: 1px solid gray;
}
QPushButton:pressed
{
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #B0E0E6, stop: 1 #B0C4DE);
}</string>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: rgb(70,130,180);
	color: white;
	border: 1px solid gray;
}
QPushButton:pressed
{
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #B0E0E6, stop: 1 #B0C4DE);
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_sometext">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер выводимого пакета:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_packetView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>401</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_search">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество пакетов:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Минимальная длина паета: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Максимальная длина пакета:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Средняя длина пакета:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_number">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_minLength">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_maxLength">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_averageLength">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_empty">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>371</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	
	background-color: rgb(0, 0, 127);
	color: white;
	border: 1px solid gray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
